### PR TITLE
v0.0.5 Bug Fixes

### PR DESCRIPTION
**Pull request by kgang**
Fix tool use

### CODE REVIEW

File: src/concrete-core/concrete/tools/utils.py [hunk 1]
```diff
@@ -18,11 +18,11 @@ def invoke_tool(tool: Tool):
     Throws TypeError if the parameters are wrong.
     """
     tool_name = tool.tool_name
-    tool_function = tool.tool_method
+    tool_function = "".join([c for c in tool.tool_method if c.isalnum()])
```
Comment (MichaelDeng03) [inline src/concrete-core/concrete/tools/utils.py:21]: Why was this necessary?

Comment (kgang) [reply src/concrete-core/concrete/tools/utils.py:21]: Sometimes it'll output `function()` vs `function`

File: src/concrete-core/concrete/operators.py [hunk 2]
```diff
@@ -292,6 +292,7 @@ def summarize_from_children(
 {joined_summaries}"""  # noqa
 
 
+# Dedicated to Michael Deng
```
Comment (MichaelDeng03) [inline src/concrete-core/concrete/operators.py:295]: <3

**Review by MichaelDeng03 — APPROVED** [review]
